@xen-orchestra/lite/src/components/infra/InfraHostItem.vue

```diff
@@ -1,5 +1,12 @@
 <template>
-  <li v-if="host" class="infra-host-item">
+  <li
+    v-if="host"
+    class="infra-host-item"
+    v-tooltip="{
+      content: host.name_label,
+      disabled: isTooltipDisabled,
```

Comment:
It is not possible to have `disabled: isTooltipDisabled` by default, since it will be the most common case while keeping the possibility of overwriting it?

Reply:
We could add a tooltip on various HTML elements.

We can not be sure that these element could have text ellipsis nor that the inner selector would be `.text`.

I think it's too abstract to be a default.